docs/content/en/docs/clustermgmt/cluster-upgrades/vsphere-and-cloudstack-upgrades.md

```diff
@@ -15,6 +15,8 @@ The upgrade command also upgrades core components of EKS Anywhere and lets the u
 
 **Upgrades should never be run from ephemeral nodes (short-lived systems that spin up and down on a regular basis). If an upgrade fails, it is very important not to delete the Docker containers running the KinD bootstrap cluster. During an upgrade, the bootstrap cluster contains critical EKS Anywhere components. If it is deleted after a failed upgrade, they cannot be recovered.**
 
+**Please make sure to have sufficient available DHCP addresses from VMs counts and maxSurge config. To create, each VM needs 1 IP. To upgrade/scale, cp/works needs 1 extra ip, each external etcd needs 1 extra ip, plus max surge. The ips in use are already taken, please free up required ips before starting the upgrade process. For example, if you need to create a cluster with 1 control plane node, 2 worker node and 3 external etcd, you'll need at least 1+2+3=6 ips. To upgrade the same cluster with maxSurge set to 2, you'll need 1+3+2=6 extra ips.
```

Comment:
For consistency and as `IP` is a proper noun, please capitalize all instances of `ip`. (`ip`->`IP`)
Additional suggestions:
- `DHCP addresses` -> `IP addresses in your DHCP pool`
- `from VMs counts and maxSurge config. To create,` -> `for all lifecycle operations. For the create operation,`
- `To upgrade/scale, cp/works needs 1 extra ip, each external etcd needs 1 extra ip, plus max surge.` -> `For upgrade and scale operations, each control plane, worker, and ETCD node need 1 extra IP plus IPs equal to the number used for maxSurge`
- `The ips in use are already taken, please free up required ips before starting the upgrade process` -> `After calculating the required IPs, make sure you environment has enough available IPs.`
- `3 external etcd` -> `3 external ETCD nodes`
- `least 1+2+3=6 ips.` - > `least 6 (1+2+3) IPs available.`
- `need 1+3+2=6 extra ips.` -> `need 6 (1+3+2) available IPs.`